Vagrantfile

```diff
@@ -72,6 +72,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     "rebuild_yum_cache" => false,
     "cpus"              => ENV['OPENSHIFT_NUM_CPUS'] || 2,
     "memory"            => ENV['OPENSHIFT_MEMORY'] || 2048,
+    "fixup_networking"  => true,
```

Comment:
My intent was to have the option overrideable solely via the json configuration file, but if you want an env var I can add one.